wsagent/agent/src/main/resources/org.eclipse.che.ws-agent.script.sh

```diff
@@ -54,9 +58,12 @@ fi
 MACHINE_TYPE=$(uname -m)
 
 mkdir -p ${CHE_DIR}
-${SUDO} mkdir -p /projects
-${SUDO} sh -c "chown -R $(id -u -n) /projects"
-${SUDO} chmod 755 /projects
+
+if is_current_user_sudoer; then
+    ${SUDO} mkdir -p /projects
```

Comment:
is it ok if /projects is only created for sudoers ?

Reply:
I think by the time this script runs `/projects` is already there since wither Docker or OpenShift creates it when the container starts/is created.

Reply:
@benoitf in any case if the user is not a sudoer he cannot create the folder `/projects/`.

Reply:
@l0rd user can be `root` user